src/chunks/metadata.rs

```diff
@@ -12,7 +13,30 @@ use crate::io::{Error, ErrorKind, ReadMetadata, Result};
 ///
 /// finalfusion metadata in TOML format.
 #[derive(Clone, Debug, PartialEq)]
-pub struct Metadata(pub Value);
+pub struct Metadata {
+    data: Value,
+}
+
+impl Metadata {
+    /// Construct new `Metadata`.
+    pub fn new(data: Value) -> Self {
```

Comment:
I think a `From` counterpart would be nice here as well.

Reply:
Tried to implement `TryFrom<V> where V: Serialize`, but that fails because of blanket impls...
~~~
  --> src/chunks/metadata.rs:43:1
   |
43 | impl<V> TryFrom<V> for Metadata where V: Serialize {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: conflicting implementation in crate `core`:
           - impl<T, U> std::convert::TryFrom<U> for T
             where U: std::convert::Into<T>;
~~~